client/src/pages/ViewRequestStatus/ButtonComponent.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const ButtonComponent = (props) => {
```

Comment:
I'm not sure about this buttoncomponent - might be easier to just do a button - but let's find out :)